server/coordinator/scheduler/scheduler_manager.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package scheduler
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sync"
+	"time"
+
+	"github.com/CeresDB/ceresmeta/pkg/log"
+	"github.com/CeresDB/ceresmeta/server/cluster"
+	"github.com/CeresDB/ceresmeta/server/coordinator"
+	"github.com/CeresDB/ceresmeta/server/coordinator/procedure"
+	"github.com/CeresDB/ceresmeta/server/storage"
+	"go.uber.org/zap"
+)
+
+const (
+	schedulerInterval = time.Second * 5
+)
+
+// Manager used to manage schedulers.
+// Each registered scheduler will generate a procedure if the cluster s
```

Comment:
The comments seem not integrate.
```suggestion
// Manager manages all schedulers.
//
// Each registered scheduler will generate a procedure if the clusters ...
```